src/common/mailing.ts

```diff
@@ -156,6 +156,8 @@ export const getContactIdByEmail = async (email: string) => {
 const LIST_REGISTERED_USERS = '85a1951f-5f0c-459f-bf5e-e5c742986a50';
 // Id of "Weekly recap" contact list on SendGrid
 const LIST_MARKETING_EMAILS = '53d09271-fd3f-4e38-ac21-095bf4f52de6';
+// Id of "Drop campaign" contact list on SendGrid
+export const LIST_DRIP_CAMPAIGN = '919b5624-ccc9-46df-960d-f6a60eb8241b';
```

Comment:
A bit more explicit, wdyt?

```suggestion
export const LIST_SQUAD_DRIP_CAMPAIGN = '919b5624-ccc9-46df-960d-f6a60eb8241b';
```